var/spack/repos/builtin/packages/protobuf/package.py

```diff
@@ -138,7 +139,10 @@ def install(self, spec, prefix):
         args = self.cmake_args()
         args.extend(std_cmake_args)
 
-        source_directory = join_path(self.stage.source_path, "cmake")
+        if self.spec.satisfies("@:3"):
+            source_directory = join_path(self.stage.source_path, "cmake")
+        else:
+            source_directory = self.stage.source_path
```

Comment:
Is this change still necessary now that we're using 3.21.7?

Reply:
@adamjstewart yes, see the contents of `CMakeLists.txt` in `cmake/`:
> Calling of cmake with source directory set to "cmake" subdirectory of Protocol Buffers project is deprecated. Top-level directory of Protocol Buffers project should be used instead.